data/logs_model/table_logs_model.py

```diff
@@ -279,15 +282,19 @@ def load_logs():
                 )
 
                 logs_query = model.log.get_logs_query(
-                    namespace=namespace_id,
+                    namespace_id=namespace_id,
                     repository=repository_id,
                     start_time=current_start_datetime,
                     end_time=current_end_datetime,
                 )
                 logs = list(logs_query)
                 for log in logs:
+                    assert isinstance(log, BaseModel)
```

Comment:
Why is this check needed?

Reply:
Just to be absolutely sure we're getting row instances out of the query.